docs/install/remote.md

```diff
@@ -144,19 +124,50 @@ project.set_secrets(file_path=env_file)
 function.set_envs(file_path=env_file)
 ```
 
-### Load the configuration and credential environmental variables from the command line
+### Using MLRun SDK or CLI
+
+**You can set the environment using MLRun SDK, for example:**
+
+```python
+# Use local service
+mlrun.set_environment("http://localhost:8080", artifact_path="./")
+# Use remote service
+mlrun.set_environment("<remote-service-url>", access_key="xyz", username="joe")
+```
+
+Explanation:
+
+1. Set `MLRUN_DBPATH` to the URL of the remote MLRun database/API service:
+
+    ```ini
+    MLRUN_DBPATH=<URL endpoint of the MLRun APIs service endpoint; e.g., "https://mlrun-api.default-tenant.app.mycluster.iguazio.com">
+    ```
+
+2. If the remote service is on an instance of the Iguazio MLOps Platform (**not relevant for MLRun CE**), set the following environment variables as well:
+
+    ```ini
+    V3IO_USERNAME=<username of a platform user with access to the MLRun service>
+    V3IO_ACCESS_KEY=<platform access key>
+    ```
+
+    You can get the platform access key from the platform dashboard: select the user-profile picture or icon from the top right corner of 
+    any page, and select **Access Keys** from the menu. In the **Access Keys** window, either copy an existing access key or create a new 
+    key and copy it. Alternatively, you can get the access key by checking the value of the `V3IO_ACCESS_KEY` environment variable in a web-shell or Jupyter Notebook service.
+
+
+**You can also set the environment using MLRun CLI, for example:**
 
 1. Create an env file similar to the example, with lines in the form KEY=VALUE, and comment lines starting with "#".
 2. Use `--env-file <env file path>` in mlrun run/build/deploy/project CLI commands to load the config and credential env vars from file.
-2. Set the `MLRUN_ENV_FILE=<env file path>` env var to point to a default env file (which will be loaded on import).
+3. Set the `MLRUN_ENV_FILE=<env file path>` env var to point to a default env file (which load on import). If the MLRUN_DBPATH points to a remote iguazio cluster and the V3IO_API and/or V3IO_FRAMESD vars are not set, they are inferred from the DBPATH.
    If the `MLRUN_DBPATH` points to a remote iguazio cluster and the `V3IO_API` and/or `V3IO_FRAMESD` vars are not set, they will be inferred from the DBPATH.
```

Comment:
delete:
If the `MLRUN_DBPATH` points to a remote iguazio cluster and the `V3IO_API` and/or `V3IO_FRAMESD` vars are not set, they will be inferred from the DBPATH